ddtrace/tracer/span.go

```diff
@@ -170,8 +165,15 @@ func (s *span) setTagError(value interface{}, cfg *errorConfig) {
 	}
 }
 
-// takeStacktrace takes stacktrace
+// defaultStackLength specifies the default maximum size of a stack trace.
+const defaultStackLength = 20
```

Comment:
NIT: Go uses a depth of 32 for most profiling use cases and 100 for `debug.Stack()`. I'd probably go with 32 to lower the risk of truncating stacks.

Reply:
Makes perfect sense. Done.